spec/system/support/staff_invite_spec.rb

```diff
@@ -0,0 +1,96 @@
+require "rails_helper"
+
+RSpec.describe "Staff support", type: :system do
+  include CommonSteps
+
+  it "allows inviting a user" do
```

Comment:
```suggestion
  scenario "Initial support user invites Staff" do
```
(with a filename along similar lines)